tests/admin/posts/DELETE/test_delete_post.py

```diff
@@ -0,0 +1,63 @@
+import allure
+
+from libs.assertions import Assertions
+from libs.api.admin.post_admin import PostAdmin
+from libs.models.post import Post
+from text_resources.post import *
+from text_resources.common import *
+from utils.helpers import get_key
+
+
+PostAPI = PostAdmin()
+
+@allure.epic('CMA - Posts')
+@allure.story('CMA: DELETE - Allowed customer to delete posts')
+class TestDeletePost:
+    def test_delete_post_by_valid_id_return_no_content_successfully(self, make_post):
+        # Arrange: Create a post
+        post = Post(title=PostAPI.fake.sentence())
+        create_post_res, id = make_post(post)
+        Assertions.created(create_post_res)
+
+        # Act
+        del_post_res = PostAPI.delete_post_by_id(id)
+
+        # Assertion
+        Assertions.no_content(del_post_res)
+        get_deleted_post_res = PostAPI.get_post_by_id(id)
+        Assertions.not_found_with_message(get_deleted_post_res, 
+                                          message=not_found_cannot_read_post_msg, 
+                                          context=not_found_post_context, 
+                                          type=not_found_error_type)
+
+    def test_delete_post_by_valid_slug_return_error_message(self, make_post):
```

Comment:
convetion: .....when.....should

```suggestion
    def test_delete_post_when_delete_by_valid_slug_should_return_error_message(self, make_post):
```

Reply:
@longle91 E sửa lại rồi ạ